src/services/questions.js

```diff
@@ -0,0 +1,33 @@
+import { backendClient } from './http';
+
+export const postQuestion = async (question, order) => {
+  try {
+    const response = await backendClient.post('/questions', {
+      data: {
+        text: question,
+        type: 'text',
```

Comment:
Type shouldn't be hardcoded, we should have a dropdown in the form for choosing the question type